src/Microsoft.AspNet.Mvc.Core/ActionResults/ViewExecutor.cs

```diff
@@ -70,7 +72,14 @@ public static class ViewExecutor
             {
                 try
                 {
-                    var viewContext = new ViewContext(actionContext, view, viewData, tempData, writer);
+                    var viewContext = new ViewContext(
+                        actionContext,
+                        view,
+                        viewData,
+                        tempData,
+                        writer,
+                        htmlHelperOptions);
```

Comment:
Yeah it made more sense the more I read the review. I think I'm mostly ok with it as is.